### PR TITLE
fix: Issue #457

### LINKED ISSUE #457: No warning is shown when you click on cancel button while making a transfer .
**Summary:**

No warning toast is shown when you try to cancel a transfer either third party or direct transfer.In case the customer by mistake clicks on cancel .

**Steps to reproduce:**

Go to navigation ->Transfer or Third Party transfer->Fill the details ->Click on review ->Then click on cancel. 

**Expected behavior:**

It should open a dialog box or show a snackbar with message "Are you sure ?,All the data will be lost if you leave this page " .

**Observed behavior:**
No such dialog box or snackbar was there on clicking transfer .It's simply cancel .

**Device and Android version:**
Moto G3(6.0.1)

### PR DESCRIPTION
**Pull request by luckyman20**
Fixes #457 

@therajanmaurya Please review it!

### CODE REVIEW

File: app/src/main/java/org/mifos/selfserviceapp/utils/Toaster.java [hunk 1]
```diff
@@ -38,6 +38,13 @@ public static void show(View view, int res, int duration) {
         show(view, MifosSelfServiceApp.getContext().getResources().getString(res), duration);
     }
 
+    public static void cancelTransfer(View view, String text, String buttonText,
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/selfserviceapp/utils/Toaster.java:41]: @luckyman000786, for now, seems good, I will think about better implementation later but this is good for now. thanks

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -413,6 +413,8 @@
     <string name="total_loan_savings_description">Above Loan and Savings accout balance are
         calculated on behalf of your all savings and loan accounts.
     </string>
+    <string name="cancel_transfer">Are you sure you want to cancel the transfer?</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values/strings.xml:416]: @luckyman000786 please add the string in all strings.xml, whenever you add a new string. Here is a medium blog(https://medium.com/@therajanmaurya/add-localization-string-in-application-supported-all-languages-using-keyboard-shortcut-807e0a388672). I have written, have a look

File: app/src/main/res/values-hi/strings.xml [hunk 3]
```diff
@@ -380,6 +380,8 @@
         <item>ऋण खाते के लिए भुगतान करने के लिए, खाता अनुभाग पर जाएं, ऋण चुनें और आवश्यक ऋण खाता खोलें
             और भुगतान करें विकल्प पर क्लिक करें।</item>
     </string-array>
+    <string name="cancel_transfer">क्या आप वाकई हस्तांतरण रद्द करना चाहते हैं?</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values-hi/strings.xml]: @luckyman000786  please add the string in all strings.xml, whenever you add a new string. Here is a medium blog(https://medium.com/@therajanmaurya/add-localization-string-in-application-supported-all-languages-using-keyboard-shortcut-807e0a388672). I have written, have a look

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by luckyman20** [comment]
@therajanmaurya I have made the changes as per your recommendations

**Review by therajanmaurya — APPROVED** [review]

**Comment by therajanmaurya** [comment]
Good, Thanks for sending PR.

**Pull request merged**